### ISSUE TITLE
APIComparer not showing changes to interfaces

### ISSUE THREAD

**Issue by bording** [ParticularLabs/APIComparer#69]
While comparing the changes in the latest RabbitMQ client, it appears that APIComparer is missing some of the changes that were made:

http://apicomparer.particular.net/compare/RabbitMQ.Client/3.6.1...3.6.2

Here is the comparison of what all was changed between 3.6.1 and 3.6.2:

https://github.com/rabbitmq/rabbitmq-dotnet-client/compare/rabbitmq_v3_6_1...rabbitmq_v3_6_2

For example, there was a new property added to `IConnection` but that isn't showing up in the comparison. Since changes to an interface are always going to break implementors, I expected to see it listed.

**Comment by andreasohlund** [comment]
We haven't been showing additions since we always considered them non breaking changes. But interfaces is interesting since additions would be be non breaking for consumers but breaking for implementors. @ParticularLabs/apicomparer thoughts?

**Comment by bording** [comment]
Given that one scenario around public interfaces is allowing you to provide your own implementation of the interface, I'd vote for including them as a potential break.

**Comment by SeanFeldman** [comment]
Tend to agree with Brandon.

**Comment by SimonCropp** [comment]
yeah i think it is a breaking change. that and additional abstract members

**Comment by andreasohlund** [comment]
Makes sense to me, that was a nice find. Always had the mental model that "adding is always safe", thanks for the wake up call @bording
